packages/ra-ui-materialui/src/list/datagrid/DatagridConfigurable.stories.tsx

```diff
@@ -46,17 +47,19 @@ const AuthorField = () => <TextField label="Author" source="author" />;
 
 const theme = createTheme();
 
-const Wrapper = ({ children }) => (
+const Wrapper = ({ children, queryClient = new QueryClient() }) => (
     <StoreContextProvider value={memoryStore()}>
         <ThemeProvider theme={theme}>
             <PreferencesEditorContextProvider>
-                <TestMemoryRouter>
-                    <Inspector />
-                    <Box display="flex" justifyContent="flex-end">
-                        <InspectorButton />
-                    </Box>
-                    <Box p={2}>{children}</Box>
-                </TestMemoryRouter>
+                <QueryClientProvider client={queryClient}>
```

Comment:
This is because the buttons (EditButton here) requires react-query to check access rights